cmd/common.go

```diff
@@ -29,6 +31,12 @@ func promptMFAtoken(messagePrefix string, logger *log.Logger) string {
 			logger.Println("MFA token must be 6 digits. Please try again.")
 			continue
 		}
+
+		if _, found := globalMFACodeMap[t]; found {
+			logger.Println("MFA token has already been used. Please try again.")
+			continue
+		}
```

Comment:
This reads nicely and I appreciate that it follows the same rules as the other validators. Consistently decrementing the number of attempts for invalid or reused inputs feels reasonable.